lib/spack/spack/stage.py

```diff
@@ -400,7 +400,7 @@ def source_path(self):
         """Returns the well-known source directory path."""
         return os.path.join(self.path, _source_path_subdir)
 
-    def fetch(self, mirror_only=False):
+    def fetch(self, mirror_only=False, manual_download=False):
```

Comment:
As we are at it, can we improve this docstring?

Reply:
Took a pass at that.